doc/docs/master/how-tos/monitor-job-progress.md

```diff
@@ -0,0 +1,52 @@
+# Monitor Job Progress
+
+After a pipeline starts a job, you can run one of the following commands
+to monitor its status:
+
+* `pachctl list pipeline`
+
+  This command shows all pipelines that run in your cluster
+  and their current status. Jobs for each pipeline will finish with either
+  `success` or `fail` status.
+
```

Comment:
I think we want to expand on more detail here. particularly, the STATE / LAST JOB field is important. 

STATE is the state of the pipeline. A pipeline state is "running" if the worker pods are spun up. That does _not_ necessarily mean that the pipeline is actively processing a job. A "failed" state indicates that the pipeline workers were not able to be created. This is mostly often something at the kubernetes or infrastructure level such as an incorrect image name or networking failures. 

LAST JOB indicates exactly that -- the status of the most recent job for that pipeline.

Reply:
added.

Reply:
> STATE is the state of the pipeline. A pipeline state is "running" if the worker pods are spun up. That does not necessarily mean that the pipeline is actively processing a job. A "failed" state indicates that the pipeline workers were not able to be created. This is mostly often something at the kubernetes or infrastructure level such as an incorrect image name or networking failures.

I think your version misses a few important items that I have in mine. Specifically:

> The running state does not necessarily mean
  that the pipeline is actively running

Yes it does. It does mean the pipeline is running, but it does _not_ mean it is actively processing a job. I think your phrasing should be more precise. 

Second, I think we should note that "failed" is a terminal state for a pipeline -- meaning the user will need to update it in order to get it out of that state. 

Also note, as JD demoed last week, in 1.11 (including the current beta1) there is a new state for pipelines called `crashing`. `crashing` indicates that the worker that is failing, but for potentially transient reasons. The two most common reasons for `crashing` are image pull failures (e.g. incorrect image name or registry credentials) or scheduling failures (e.g. not enough resources on your k8s cluster).

Reply:
Added. Will also need to update the troubleshooting section with this.